From 7bc056786bf9b5f1df90b3f6a7fa6506d41965da Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Wed, 31 Jul 2019 22:58:48 +0200
Subject: [PATCH] modules: update OpenWrt

df53824f46 kernel: bump 4.14 to 4.14.134
76d1e8a0c2 kernel: bump 4.9 to 4.9.186
ef9c13fb5d imx6: bump SDMA firmware to 3.5
41e3f12e00 imx6: bump sdma firmware to 3.4
f51e2d031e mac80211: brcm: improve brcmfmac debugging of firmware crashes
95745e26b3 mac80211: brcm: update brcmfmac 5.4 patches
65a405382b omcproxy: fix compilation on little-endian CPUs
a68be42f99 scripts: ipkg-make-index.sh: dereference symbolic links
8231f67218 mac80211: brcmfmac: backport fixes from kernel 5.4
627bb0b8dc busybox: strip off ALTERNATIVES spec
e6af9c017b opkg: bump to version 2019-06-14
33e7beeb31 base-files: Fix path check in get_mac_binary
687977bfc9 kernel: bump 4.14 to 4.14.132
6ee6c97ded base-files: Really check path in get_mac_binary
aced9de9a4 wireguard: bump to 0.0.20190601
3bbd16da46 ramips: fix mt7620 pinmux for second SPI
b84f761d91 OpenWrt v18.06.4: revert to branch defaults
f6429577c5 OpenWrt v18.06.4: adjust config defaults

Compile-tested: ar71xx-generic, ipq40xx, mpc85xx-generic, ramips-mt7621
Runtime-tested: mpc85xx-generic (WDR4900v1), ramips-mt761 (DIR-860L-B1)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 67c2d5fc..e45a1c6d 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing luci gluon'
 
 OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-18.06
-OPENWRT_COMMIT=ef686b72923792d6db852a752a07649fe38eca6b
+OPENWRT_COMMIT=df53824f462d7205225bf5a8b51e872b27903e14
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
-- 
2.17.1

